src/test/java/com/coherent/task60/LoginPage.java

```diff
@@ -36,7 +31,12 @@ public By getPasswordField() {
         return passwordField;
     }
 
+    public WebDriver getDriver() {
+        return driver;
+    }
```

Comment:
This method should be removed. If you need to get driver you can call method from singleton class

Reply:
Got it. Removed